### ISSUE TITLE
Validation not triggered when using a custom JSON.NET CustomCreationConverter

### ISSUE THREAD

**Issue by rolandoldengarm** [JeremySkinner/FluentValidation#1026]
### System Details

- FluentValidation version: 7.3.0
- Web Framework version (eg ASP.NET Core 2.1, MVC 5, WebApi 2. Delete if not applicable):
ASP.NET Core 1.1

### Issue Description
Everything was working fine, but after I've implemented a custom CustomCreationConverter for my objects, the validator is no longer triggered.
So, in the Controller method; ModelState.IsValid is always true.

I could not find anything on Google about this. Should I trigger the validation manually in my CustomCreationConverter? But how do I set the ModelState.IsValid to false if it's not valid?

So, my code:

```
{
// Uncommenting this causes the validation to stop being executed:
// [JsonConverter(typeof(AssetModelConverter))]
public class AssetModel
...
}

public class AssetModelConverter : CustomCreationConverter<AssetModel>
{
...
}

public class AssetModelValidator: AbstractValidator<AssetModel>
{
    public AssetModelValidator()
    {
    // validation rules
    }
}
```

**Comment by JeremySkinner** [comment]
I wouldn't expect you to need to trigger validation manually, but that being said ASP.NET Core 1.1 isn't actually supported by FluentValidation (asp.net core 2.1 is the lowest supported version since FluentValidation 7.3, with FluentValidation 8.1 supporting asp.net core 2.2), so in this case you may be better off running validation manually. 

> But how do I set the ModelState.IsValid to false if it's not valid?

`ModelState.IsValid` is automatically returns false if the ModelState colletion contains any validation errors.

**Comment by rolandoldengarm** [comment]
Thanks @JeremySkinner for your reply! Yep it is very weird, I would not expect a JSON Converter to interfere with validation.
We're stuck on ASP.NET Core 1.1 unfortunately. So I'll keep running the validation manually.
